bindings/java/pom.xml

```diff
@@ -24,11 +24,31 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
+        <version>29</version>
```

Comment:
What's this version for?

Reply:
We add Apache POM as the parent to get all default setups - see https://maven.apache.org/pom/asf/